### PR TITLE
Draft: highlightbox formating

### PR DESCRIPTION
**Pull request by AndrewB9lab**
Documentation content update for All content

Please do not merge!

### Change scope

The changes in this Pull Request include (please tick all that apply):

* [ ] Small language/grammar fixes
* [ ] Small content fixes 
* [ ] Addition of new content
* [ ] Sample code/command updates
* [X] Platform fixes
* [ ] Other

### CODE REVIEW

File: course-ida/quiz-week1.md [hunk 1]
```diff
@@ -11,12 +11,13 @@ You have reached the end of week 1. Time for a check up to see how much you have
...
 **Please make sure you enter the same email address used to sign up for this course when submitting the quiz.**
 
-
 Follow this link: [Week 1 quiz](https://hr.gs/ida-week1-quiz), to submit a pop quiz covering general questions on Cosmos and its main concepts. The quiz contains 38 questions and you have 120 minutes to complete it.
 
 <HighlightBox type="note">
 
 You need to make sure to submit the quiz by **May 20th** latest! In case you cannot make sure to hit this due date, please reach out to us!
+<br></br>
+Implemented highlight box fixes for Academy course, Ch1-3
```
Comment (xavierlepretre) [inline course-ida/quiz-week1.md:20]: Was this piece intended?

File: academy/4-ibc/channels.md [hunk 2]
```diff
@@ -35,6 +35,7 @@ As mentioned above, channels are payload agnostic. The application modules sendi
 <HighlightBox type="info">
 
 An **ordered channel** is _a channel where packets are delivered exactly in the order in which they were sent_.
+<br></br>
 An **unordered channel** is _a channel where packets can be delivered in any order_, which may differ from the order in which they were sent.
```
Comment (xavierlepretre) [inline academy/4-ibc/channels.md:39]: Should this be a list instead?

File: academy/1-what-is-cosmos/atom-staking.md [hunk 3]
```diff
@@ -46,7 +46,7 @@ When Keplr is installed it should be available in the list of extensions when yo
 <HighlightBox type="warn">
 
 When you own ATOM with Keplr, **you are in charge**. There is no one you can call if _you forget your password/seed phrase or you send tokens to the wrong address_.
-
+<br></br>
```
Comment (xavierlepretre) [inline academy/1-what-is-cosmos/atom-staking.md:49]: Can something equivalent not be done during the compilation? @coldice

**Review by xavierlepretre — CHANGES_REQUESTED** [review]

File: course-ida/quiz-week1.md (hunk 1 above)
Comment (AndrewB9lab) [reply course-ida/quiz-week1.md:20]: Oops, no, definitely not

File: academy/4-ibc/channels.md (hunk 2 above)
Comment (AndrewB9lab) [reply academy/4-ibc/channels.md:39]: Sure, I'll make this change.

File: academy/1-what-is-cosmos/atom-staking.md (hunk 3 above)
Comment (coldice) [reply academy/1-what-is-cosmos/atom-staking.md:49]: It is a bit fighting against the Markdown spec - Markdown doesn't allow multiple blank lines. For compatibility, it is better to use these specific html tags instead. We were thinking about automatically converting multiple empty lines into br, but it is context-dependent.

**Comment by coldice** [comment]
Linkcheck OK

**Pull request merged**